index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home - Smart City Development Konsult Ltd</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=cocogoose">
+    <script src="https://kit.fontawesome.com/a7a68c4557.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <nav class="navbar-black">
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+      <div class="menu">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <ul class="header-menu">
+          <li class="header-menu-item"><a href="index.html">English</a></li>
+          <li class="header-menu-item"><a href="index.html">My Page</a></li>
+          <li class="header-menu-item"><a href="index.html">Login</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="navbar-white">
+      <div class="logo">
+        <a href="index.html"><img src="images/logo1.png" alt="SCDK logo"></a>
```

Comment:
- [ ] Please follow the given design in styling your navbar. The given design has to be followed in all its aspects such as 'Spacings
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/165544147-fb299cf8-bac9-4c85-9cc4-6ebd2997deea.png)
'